src/frontend/src/stores/authStore.ts

```diff
@@ -19,8 +20,8 @@ const cookies = new Cookies();
 
 const useAuthStore = create<AuthStoreType>((set, get) => ({
   isAdmin: false,
-  isAuthenticated: !!cookies.get("access_token_lf"),
-  accessToken: cookies.get("access_token_lf") ?? null,
+  isAuthenticated: !!!!cookies.get(LANGFLOW_ACCESS_TOKEN),
```

Comment:
just a question, this:
isAuthenticated: !!cookies.get(LANGFLOW_ACCESS_TOKEN) 
is not supposed to have the same behavior?

Reply:
yes, i don't know why i did this lol

Reply:
i will fix it now